demos/storybook/stories/user-menu/with-full-config.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { text } from '@storybook/addon-knobs';
+
+const items = [
+    {
+        title: 'Account',
+        icon: 'settings',
+    },
+    {
+        title: 'Contact Us',
+        icon: 'mail',
+    },
+    {
+        title: 'Log Out',
+        icon: 'logout',
+    },
+];
+
+export const withFullConfig = (): any => ({
```

Comment:
This demo is identical to the 'with a menu header' story (see comment there). React has the avatar and menu props slit into two sections, but since there's only three on the angular side that is probably overkill.

Reply:
If I make the positions an @Input then then these new knobs should slightly beef up this story to be more exciting.